pytest.ini

```diff
@@ -14,7 +14,7 @@ python_files = test*.py
 
 # Because we provide our own collection logic, disable the default
 # python collector by giving it empty patterns to search for.
-python_classes =
+python_classes = *Suite
```

Comment:
Perhaps it would be better to explicitly check via e.g. isinstance checks in the custom collector?

Reply:
The custom collector is for the data-driven tests. It doesn't know about tests derived from TestCase, and I think it might be awkward to make it handle the non-TestCase, non-data-driven tests. I am still thinking about the right way to collect tests, but I prefer to work on it in following PRs.

Reply:
What is the reason for this change?

Reply:
The comment above this line is no longer up-to-date.